pkg/apis/flux/v1alpha1/types.go

```diff
@@ -24,6 +24,18 @@ type FluxConfig struct {
 	// If provided, "Source" must also be provided.
 	// +optional
 	Kustomization *Kustomization `json:"kustomization,omitempty"`
+
+	// AdditionalSecretResources to sync to the shoot.
```

Comment:
Can you document the behavior? E.g., only create if it doesn't exist, no update, but cleanup, ...